### ISSUE TITLE
Exchange Details page missing button

### ISSUE THREAD

**Issue by oanamangiurea** [SelfKeyFoundation/Identity-Wallet#1117]
If I haven't unlocked the marketplace yet and I go to an exchange details page, we are supposed to still show the Sign Up button, but disabled and with a message under, as in the design https://projects.invisionapp.com/d/main#/console/15222689/347983086/preview 

Right now that area is missing entirely

**Comment by andregoncalves** [comment]
@rodrigopavezi I'm dealing with only the part after marketplace is unlocked, this is the staking right?

**Comment by rhdelima** [comment]
@eduardosmoura ,

I checked on latest build (dev_2019.05.03.20) and PR build (dev_2019.05.03.18) but I still can't see the Sign Up button.

![image](https://user-images.githubusercontent.com/10294014/57221010-13890a00-7030-11e9-8f55-91e66b9eaab8.png)

**Comment by eduardosmoura** [comment]
@rhdelima looks like it failed while building on last Friday. I think @rodrigopavezi will push a new build because we merged several PRs today. Could you please wait a moment and test it again with today's version?

**Comment by CortecKYC** [comment]
@rodrigopavezi , please help unblock Ralph ^^

**Comment by rhdelima** [comment]
Closing this one. Sign Up button is now shown even if marketplace is still locked but disabled. Another ticket was opened to address issue for Sign Up button not enabled after staking

![image](https://user-images.githubusercontent.com/10294014/57573009-b8eb1600-7454-11e9-8b17-91cfc2ed2f59.png)
